doc.go

```diff
@@ -20,6 +20,7 @@
 // The different modules of the GleSYS API are available on the client.
 // For example:
 //
+//     client.Emails.Overview(...)
```

Comment:
```suggestion
//     client.EmailDomains.Overview(...)
```

Reply:
Ah, totally missed that one, thank you for updating it for me 👍